js/piwik.js

```diff
@@ -3466,64 +3466,60 @@ if (typeof window.Matomo !== 'object') {
                 // note: there might be negative values because of browser bugs see https://github.com/matomo-org/matomo/pull/16516 in this case we ignore the values
                 var timings = '';
 
-                if (performanceAlias && performanceAlias.timing && performanceAlias
-                    && performanceAlias.timing.connectEnd && performanceAlias.timing.fetchStart) {
+                var performanceData = (typeof performanceAlias.getEntriesByType === 'function') && performanceAlias.getEntriesByType('navigation') ? performanceAlias.getEntriesByType('navigation')[0] : performanceAlias.timing;
```

Comment:
In the previous code we check that `performanceAlias != null`, do we still need to do this?

Reply:
makes sense. I'll update the code